gandalf.js

```diff
@@ -1,130 +1,133 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!